app/pages/home/home.tsx

```diff
@@ -52,15 +50,14 @@ export const Home: FC = () => {
   useInterval(() => {
     if (!address) return;
     pendingTxs.forEach(tx => void checkIfPendingTxIsComplete(safelyFormatHexTxid(tx.txId)));
-    // dispatch(getAddressTransactions(address));
-    // dispatch(getAddressDetails(address));
+    dispatch(getAddressTransactions(address));
+    dispatch(getAddressDetails(address));
   }, 5_000);
```

Comment:
Should this be inside of `useEffect`? Also, should we be clearing the interval once the component is unmounted?

Reply:
This is to poll to requests, so we do want it in an interval. Clearing of interval is handled by the hook.

Reply:
Ah, duh!